File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/OppfyltVedSammenlegging.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.kode.AvtalelandCti;

import java.io.Serializable;

public class OppfyltVedSammenlegging implements Serializable {

    

    /**
     * Angir om kriteriet er oppfylt eller ikke.
     */
    boolean oppfylt;

    /**
     * Angir avtaleland.
     */
    AvtalelandCti avtaleland;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/OppfyltVedSammenlegging.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.enum.AvtaleLandEnum
import java.io.Serializable

class OppfyltVedSammenlegging : Serializable {
    /**
     * Angir om kriteriet er oppfylt eller ikke.
     */
    var oppfylt = false

    /**
     * Angir avtaleland.
     */
    var avtalelandEnum: AvtaleLandEnum? = null
}